_sass/leaderboard.scss

```
@@ -29,6 +29,31 @@
   }
 }
 
+.leaderboard.dark {
+  
+  a {
+    color: white;
+  }
+
+  tr:nth-child(2n + 1) {
```

Here we can just set the color for all `tr`'s:

```suggestion
  tr {
```